src/seedu/addressbook/storage/Storage.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.storage;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.storage.StorageFile.StorageOperationException;
+
+/**
+ * Necessary methods from StorageFiles for use in Logic.
+ */
+public abstract class Storage {
```

Comment:
Try to come up with better description as this class might be used with other child classes, not only StorageFiles.